x-pack/plugins/ml/public/application/services/field_format_service_provider.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { type MlFieldFormatService, FieldFormatService } from './field_format_service';
+import type { MlIndexUtils } from '../util/index_utils_provider';
+import type { MlApiServices } from './ml_api_service';
+
+export function fieldFormatServiceProvider(
```

Comment:
Suggest to rename this to `fieldFormatServiceFactory` and the file to `field_format_service.ts` to be in line with how we renamed the time buckets service.